perllib/FixMyStreet/Cobrand/UKCouncils.pm

```diff
@@ -501,6 +501,21 @@ sub updates_disallowed {
     return $self->next::method(@_);
 }
 
+# Report if cobrand denies updates by user
+sub deny_updates_by_user {
+    my ($self, $is_open) = @_;
```

Comment:
I wonder if passing in the row here and then doing the is_open check might be a bit more future proof